viz/cmd/tap.go

```diff
@@ -353,13 +361,12 @@ func renderTapEvent(event *tapPb.TapEvent, resource string) string {
 		tls,
 	)
 
-	// If `resource` is non-empty, then
 	resources := ""
-	if resource != "" {
+	if wide {
 		resources = fmt.Sprintf(
-			"%s%s%s",
-			src.formatResource(resource),
-			dst.formatResource(resource),
+			" %s %s %s",
```

Comment:
i think this is adding an extra space before the next set of labels if the `src` or `dst` labels are empty and `formatResource()` returns an empty string. can we change this back to
```suggestion
			"%s%s%s",
```
and have `formatResource` add a trailing space to the string if it's non-empty, or something?